xbmc/guilib/TextureBundle.cpp

```diff
@@ -66,6 +66,12 @@ int CTextureBundle::LoadAnim(const std::string& Filename, CBaseTexture*** ppText
   return 0;
 }
 
+void CTextureBundle::Close()
+{
+  if (m_useXBT)
```

Comment:
IMO it would be safer to always call `m_tbXBT.CloseBundle()` or does that cause crashes?

Reply:
@Rechi ?

Reply:
As said, I don't see a reason why `m_tbXBT.CloseBundle() ` should only be called if `m_useXBT` is true. Closing a TextureBundle means you have to always close it's TextureBundleXBT (`m_tbXBT`) and TextureBundleXBT should (and does) take care what's needed to properly close that. It was implemented like that somewhere else is not a good reason to keep it exactly the same.

But I don't care much if it is kept like that.

Reply:
@Rechi 

If you think that this should be changed then I'll change it - after all it works perfectly well without it. Just let me know.